calico-enterprise/operations/cnx/configure-identity-provider.mdx

```diff
@@ -31,6 +31,7 @@ When configuring your cluster, you may be asked for the following inputs:
 - **OIDC authentication**: User identity is managed outside of the cluster by an OIDC authorization server.
 - **Google OIDC authentication**: User identity is managed by Google OIDC. Choose this option if you want to use GSuite groups.
 - **OpenShift authentication**: User identity is provided by the OpenShift OAuth server.
+- **LDAP authentication**: User identify is provided using Active Directory.
```

Comment:
identity*

I would say ldap server, not active directory, even though AD is the most common application of ldap.

Reply:
Fixed.